### PR TITLE
Fixing #3

### PR DESCRIPTION
**Pull request by shekhei**
fixing #3

### CODE REVIEW

File: lib/adapter.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ module.exports = (function() {
 
         var connectionObject = connections[connectionName];
         var collection = connectionObject.collections[table];
-
+	var databaseQuery = (client.connectionParameters && client.connectionParameters.database) ? " and c.table_schema = '"+client.connectionParameters.database+"' " : "";
```
Comment (hchandl) [inline lib/adapter.js]: Sorry for the nitpick, but can you replace the tab with spaces here?

**Comment by hchandl** [comment]
Good catch! Just a minor in-line comment, and then I'll be set to merge this in.

File: lib/adapter.js (hunk 1 above)
Comment (shekhei) [inline lib/adapter.js]: np!

**Comment by shekhei** [comment]
ok, new changes are pushed, merge when if its good to go!

**Comment by hchandl** [comment]
Looks good, thanks!

**Pull request merged**